drivers/platform/x86/asus-wrc.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * Asus Wireless Radio Control Driver
+ *
+ * Copyright (C) 2015 Endless Mobile Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ *
+ */
+
+#define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
+
+#include <linux/kernel.h>
+#include <linux/module.h>
+#include <linux/init.h>
+#include <linux/types.h>
+#include <linux/acpi.h>
+#include <linux/input.h>
+#include <linux/input/sparse-keymap.h>
+
+#define ASUS_WRC_MODULE_NAME "Asus Wireless Radio Control Driver"
+#define ASUS_WRC_MODULE_AUTHOR "João Paulo Rechi Vita <jprvita@gmail.com>"
+#define ASUS_WRC_MODULE_LICENSE "GPL"
+#define ASUS_WRC_MODULE_CLASS "hotkey"
+
+#define ASUS_WRC_INPUT_NAME "Asus Wireless Radio Control"
+
+#define ASUS_WRC_FILE "asus-wrc"
+
```

Comment:
You're right, I'm removing all of them except ASUS_WRC_MODULE_NAME since I use it a few times throughout the code.